### PR TITLE
Set Content-Length to 0 in XSendfile middleware #602

### PR DESCRIPTION
**Pull request by robrwo**
Also add POD documenting the configuration options.

### CODE REVIEW

**Comment by miyagawa** [comment]
Remove the option and always reset the content length to 0. This is a bug inherited from Rack's senfile middleware.

File: lib/Plack/Middleware/XSendfile.pm [hunk 1]
```diff
@@ -78,6 +80,18 @@ for the frontend configuration.
 
 Plack::Middleware::XSendfile does not set the content-type header.
 
+=head1 CONFIGURATION
+
+=over 4
+
+=item variation
+
+The header tag.  Defaults to C<X-Sendfile>.
```
Comment (miyagawa) [inline lib/Plack/Middleware/XSendfile.pm]: I don't think this is accurate.

**Comment by robrwo** [comment]
@miyagawa Changes fixed. Is this alright to merge?

**Comment by miyagawa** [comment]
Thanks!

**Comment by robrwo** [comment]
Thanks as well. Any plans to release soon?

**Pull request merged**